docs/api/ref/api.yml

```diff
@@ -232,7 +232,7 @@ paths:
             schema:
               AllOf:
               - $ref: ./requestBodies/add_or_edit_a_product.yaml
-              - $ref: ./responses/change_ref_properties.yaml
+              - $ref: ./requestBodies/change_ref_properties.yaml
```

Comment:
The file's location in the repo seems to be [`docs/api/ref/responses/change_ref_properties.yaml`](https://github.com/openfoodfacts/openfoodfacts-server/blob/ddd88a6a95dfb09bc753e49527cd732ca21d3a44/docs/api/ref/responses/change_ref_properties.yaml)

Reply:
Thank you for the review. 

You are correct that the response schema is located at [`docs/api/ref/responses/change_ref_properties.yaml`](https://github.com/openfoodfacts/openfoodfacts-server/blob/ddd88a6a95dfb09bc753e49527cd732ca21d3a44/docs/api/ref/responses/change_ref_properties.yaml).

However, the requestBody schema is in a different location: [`docs/api/ref/requestBodies/change_ref_properties.yaml`](https://github.com/openfoodfacts/openfoodfacts-server/blob/ddd88a6a95dfb09bc753e49527cd732ca21d3a44/docs/api/ref/requestBodies/change_ref_properties.yaml).

In the initial commit (6370cc3), the response schema file was missing, while the requestBody schema file was present. Unfortunately, the `api.yaml` file referenced the missing response schema file in both places, causing the validation issue.

This PR addresses the problem by:
- Adding the missing response schema file.
- Correcting the `api.yaml` references to properly distinguish between the response schema and the requestBody schema.

I hope this clarifies the changes. Let me know if there are any further adjustments needed.